lib/module.js

```diff
@@ -18,7 +18,7 @@ export default async function module(moduleOptions) {
   this.extendBuild(extendBuild.bind(this))
 
   options._input = options.input
-  options.input = resolve(options.input)
+  options.input = resolve(options.input).replace(/\//g, path.sep);
```

Comment:
This is what `resolve` function should have done, I think it's better to move inside `resolve`

Reply:
```suggestion
  options.input = resolve(options.input.replace(/\//g, path.sep));
```